README.md

```diff
@@ -27,7 +27,10 @@ npm install typechat
 You can also build TypeChat from source:
 
 ```
-npm run build
+git clone https://github.com/microsoft/TypeChat
+cd TypeChat
+pnpm install
```

Comment:
Not everyone knows `pnpm`, I'd recommend adding `pnpm` install instructions (it could seem like a typo. https://pnpm.io/cli/install